CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to USWDS web components
+
+Thank you for your interest in contributing to USWDS web components.
+
+> [!Important]
+>
+> **This repo is currently in beta and is not yet ready for community code contributions.**
+>
+> While the USWDS core team works through big decisions about architecture and style, we ask that you do not submit pull requests. Instead, we encourage you to contribute by your sharing your ideas, concerns, questions, and experiences with the USWDS core team and community.
```

Comment:
**note:** I cut this because it felt too verbose, but wanted to add it here in the notes in case there is interest in explaining why we don't want PRs at this time 
 
```suggestion
> While the USWDS core team works through big decisions about architecture and style, we ask that you do not submit pull requests. The code structure and style will continue to evolve in this phase, and we don't want you to spend your time working on a solution that might already be in progress, or solving a problem that no longer exists. Instead, we encourage you to contribute by your sharing your ideas, concerns, questions, and experiences with the USWDS core team and community.
```

Reply:
wondering if we need to say " 'big' decisions about architecture and style". Why not just "decisions about architecture and style"?

Reply:
Removed the word "big" in [03a1c5e](https://github.com/uswds/web-components/pull/45/commits/03a1c5eb80846105a69f858c122c723d1212c67b)